src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/Objection.java

```diff
@@ -86,7 +86,7 @@ public Objection build() {
     @Field("status")
     private ObjectionStatus status;
     @Field("action_code")
-    private int actionCode;
+    private Long actionCode;
```

Comment:
Should this be primitive in the object itself?

Reply:
shouldn't matter due to auto boxing, just that i'd done the validation with Long so tried to keep it consistent.

Reply:
As long as there's no danger of Long being null that should be fine.

Reply:
I hadn't thought of a null situation. I'll change to primitive then.

Reply:
What would be better a null pointer exception or a default value of 0 being assigned?

Reply:
actually, yes, a null in mongo would indicate a missing action code, perhaps companies can have no action code anyway? 0 might indicate an actual 0 code or possible missing code. I think i'll leave it as is then.

Reply:
We could have -1 - just think of what we return to the web module in this scenario.

Reply:
I guess you could still have a -1 action code in chips DB

Reply:
As long the web will handle a value of null we should be ok.

Reply:
thinking about it, we don't actually return the action code to the web, just the objection status (enum value)

Reply:
of course this action code does not get returned to the web - think of the wrong place. However, we may want to check for null where this action code intersects with creating the status that we do return to the web - added another comment below.